ecommerce/PaymentGateways/Configs/StripeConfig.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Tutor\PaymentGateways\Configs;
+
+use Ollyo\PaymentHub\Contracts\Payment\ConfigContract;
+use Ollyo\PaymentHub\Core\Payment\BaseConfig;
+
+/**
+ * StripeConfig class.
+ *
+ * This class handles the configuration for the Stripe payment gateway.
+ * It extends the BaseConfig class and implements the ConfigContract interface.
+ *
+ * @since 3.0.0
+ */
+class StripeConfig extends BaseConfig implements ConfigContract {
+
+	/**
+	 * The name of the payment gateway.
+	 *
+	 * @since 3.0.0
+	 *
+	 * @var string
+	 */
+	protected $name = 'stripe';
+
+	/**
+	 * Constructor.
+	 *
+	 * Initializes the StripeConfig object.
+	 *
+	 * @since 3.0.0
+	 */
+	public function __construct() {
+		parent::__construct();
+	}
+
+	/**
+	 * Retrieves the mode of the Stripe payment gateway.
+	 *
+	 * @since 3.0.0
+	 *
+	 * @return string The mode of the payment gateway ('test' or 'live').
+	 */
+	public function getMode(): string {
+		return 'test';
+	}
+
+	/**
+	 * Retrieves the secret key for the Stripe payment gateway.
+	 *
+	 * @since 3.0.0
+	 *
+	 * @return string The secret key.
+	 */
+	public function getSecretKey(): string {
+		return 'sk_test_51OqvUtJyMznDJzqj9g7WK5VtJT74zFM7g8ThxhA3xRi0MHdgWHo80jOVhuLTw43t7cyFNBAA1wJub0f0Y7y6zZgi00RU2ICFWB';
```

Comment:
We have to store the data into DB and fetch the data from the DB and return here.